src/pages/home/report/ReportActionsView.js

```diff
@@ -503,6 +505,9 @@ class ReportActionsView extends React.Component {
     }
 
     render() {
+        const {isDefaultRoom} = ReportUtils;
```

Comment:
it hurts. Please call these directly `ReportUtils.isDefaultRoom`. Otherwise, there is no advantage of `import * as ReportUtils from '../../../libs/reportUtils';`.

Reply:
got it.

Reply:
Ahh nope please suggest the code because we are sending the #room in the string as a prop for language.

Reply:
ok. You can break them into multiple Text nodes. 
```
<Text>This </Text>
<Text>**room**</Text>
<Text>is awesome</Text>
```

Reply:
Good catch @parasharrajat, but I'm confused why we would need to break into multiple text nodes... @MirFahad58 why does this not work?
```javascript
const isDefaultChatRoom = ReportUtils.isDefaultRoom(this.props.report);
```

Reply:
@roryabraham  @parasharrajat -  my question is if we break it into separate texts we need to break the the sentence into tow parts
right now this one is like
`beginningOfChatHistoryPrivate: ({name}) => `Este es el principio de la sala privada ${name}, invita a otros @mencionándolos.`,`

Reply:
should i do like this 
beginningOfChatHistoryPrivatePartOne
beginningOfChatHistoryPrivatePartTwo

Reply:
Yeah, I think that's fine for now.

Reply:
Before doing that try to get the two parts of the sentence in Spanish on the Slack.

Reply:
which one? separate it into two parts?

Reply:
`beginningOfChatHistoryPrivatePartOne:  Este es el principio de la sala privada`
`beginningOfChatHistoryPrivatePartOne:  room, invite others by @mentioning them.`
@parasharrajat @roryabraham what about this?